docs/alerts.md

```diff
@@ -0,0 +1,65 @@
+# Alerts
+
+<!-- TOC depthTo:2 -->
+
+- Elasticsearch cluster
+    - [ElasticsearchClusterNotHealthyRed](#ElasticsearchClusterNotHealthyRed)
+    - [ElasticsearchClusterNotHealthyYellow](#ElasticsearchClusterNotHealthyYellow)
+    - [ElasticsearchWriteRequestsRejectionJumps](#ElasticsearchWriteRequestsRejectionJumps)
+    - [ElasticsearchNodeDiskWatermarkDiskLowReached](#ElasticsearchNodeDiskWatermarkDiskLowReached)
+    - [ElasticsearchNodeDiskWatermarkDiskHighReached](#ElasticsearchNodeDiskWatermarkDiskHighReached)
+    - [ElasticsearchNodeDiskWatermarkFloodReached](#ElasticsearchNodeDiskWatermarkFloodReached)
+    - [ElasticsearchJVMHeapUseHigh](#ElasticsearchJVMHeapUseHigh)
+    - [AggregatedLoggingSystemCPUHigh](#AggregatedLoggingSystemCPUHigh)
+    - [ElasticsearchProcessCPUHigh](#ElasticsearchProcessCPUHigh)
+    - [ElasticsearchDiskSpaceRunningLow](#ElasticsearchDiskSpaceRunningLow)
+    - [ElasticsearchHighFileDescriptorUsage](#ElasticsearchHighFileDescriptorUsage)
```

Comment:
Given we link to these directly from our alerts, can we make the titles more human readable?
e.g. "Elasticsearch Cluster Health is Red", "Elasticsearch Cluster Health is Yellow", etc